### ISSUE TITLE
evaluate

### ISSUE THREAD

**Issue by learnalbert** [Zhendong-Wang/Prompt-Diffusion#17]
Hello, when you conducted metric evaluations in the iPromptDiffusion paper, how many training steps were used for the version of PromptDiffusion you utilized? Additionally, what was the specific dataset used for your testing?

**Comment by Zhendong-Wang** [comment]
It is about 5000-10000 steps for training iPromptDiffusion. The evaluation test dataset is a subset of the training data, which is not included in the training. You could refer to the ratio here: https://github.com/Zhendong-Wang/Prompt-Diffusion/blob/8ecd2d126d9f9e17b80d9655bf8ed2df2e93f660/edit_dataset.py#L23.
